docs/repositories.md

```diff
@@ -112,9 +112,27 @@ default = false
 secondary = false
 ```
 
+When a package source is defined like this, it takes precedence over [PyPI](https://pypi.org).
+
+To ignore [PyPI](https://pypi.org) completely, use `default = true`.
+
+To target a package source only on specific packages, use `secondary = true` on the package source, and specify the `source` when adding the dependency.
+In the example below, `my-lib` will be obtained from the custom package source and `urllib3` will use the [PyPI](https://pypi.org) source.
+
+```toml
+[tool.poetry.requirements]
+my-lib = { version = "*", source = "foo" }
+urllib3 = "*"
+
+[[tool.poetry.source]]
+name = "foo"
+url = "https://foo.bar/simple/"
+secondary = true
+```
```

Comment:
I'd move this below the warning for continuity. And refer to the default section. Rephrase it something like -

```md
{{% note %}}
If you prefer to disable PyPI, you may choose to set one of your package sources to be the [default](#default-package-source).
{{% /note %}}
```

And move the exampe to the secondary package source section maybe?